<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>450</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Klotski</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>280</width>
     <height>350</height>
    </size>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonFinish">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Finish</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>&amp;Game</string>
    </property>
    <addaction name="actionSelect_Level"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionRestart"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Optimal_solution"/>
    <addaction name="actionShow_High_Scores"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <widget class="QMenu" name="menuSwitch_Application_Language">
     <property name="title">
      <string>Switch Application &amp;Language</string>
     </property>
     <addaction name="actionChinese_Simplified"/>
     <addaction name="actionEnglish"/>
    </widget>
    <addaction name="actionShow_Toolbar"/>
    <addaction name="actionShow_Statusbar"/>
    <addaction name="separator"/>
    <addaction name="menuSwitch_Application_Language"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionKlotski_Handbook"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Klotski"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelect_Level"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionRestart"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetGameInfo">
   <property name="styleSheet">
    <string notr="true">#dockWidgetContentsGameInfo {
	border: 2px solid rgb(218, 218, 218);
}
#labelStepInfo {
	border: 1px solid rgb(218, 218, 218);
}
#labelStepInfo {
	background-color: white;
	font-size: 32px;
}</string>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContentsGameInfo">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelStepInfo">
       <property name="text">
        <string>StepInfo</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="HistoryView" name="historyView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/save.svg</normaloff>:/resources/icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionShow_High_Scores">
   <property name="text">
    <string>Show &amp;High Scores</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/exit.svg</normaloff>:/resources/icons/exit.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionShow_Optimal_solution">
   <property name="text">
    <string>Show &amp;Optimal solution</string>
   </property>
  </action>
  <action name="actionShow_Toolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Toolbar</string>
   </property>
  </action>
  <action name="actionKlotski_Handbook">
   <property name="text">
    <string>&amp;Klotski Handbook</string>
   </property>
  </action>
  <action name="actionAbout_Klotski">
   <property name="text">
    <string>&amp;About Klotski</string>
   </property>
  </action>
  <action name="actionChinese_Simplified">
   <property name="text">
    <string>&amp;Chinese (Simplified)</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>&amp;English</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/restart.svg</normaloff>:/resources/icons/restart.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Restart</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/open.svg</normaloff>:/resources/icons/open.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/undo.svg</normaloff>:/resources/icons/undo.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/redo.svg</normaloff>:/resources/icons/redo.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionShow_Statusbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;Statusbar</string>
   </property>
  </action>
  <action name="actionSelect_Level">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/icons/level-select.svg</normaloff>:/resources/icons/level-select.svg</iconset>
   </property>
   <property name="text">
    <string>Select &amp;Level</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HistoryView</class>
   <extends>QListView</extends>
   <header>historyview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
